vnext/ReactUWP/Views/VirtualTextViewManager.cpp

```diff
@@ -66,19 +67,26 @@ void VirtualTextViewManager::AddView(
     XamlView parent,
     XamlView child,
     int64_t index) {
-  auto span(parent.as<winrt::Span>());
-  auto childInline(child.as<winrt::Inline>());
-  span.Inlines().InsertAt(static_cast<uint32_t>(index), childInline);
+  auto spanXD = GetXamlDirect().GetXamlDirectObject(parent.as<winrt::Span>());
```

Comment:
>GetXamlDirectObject [](start = 32, length = 19)

One of the most important gains of XamlDirect is to avoid having the WinRT API projection exist at all - is there a way for "parent" here to not need to exist as a winrt::Span first?  Having that handle incurs all the costs already, so converting to XD from there isn't doing anything.